variables.tf

```diff
@@ -71,7 +77,9 @@ variable "vpc_cidr_block" {
   default     = "10.0.0.0/16"
 }
 
-# External DNS
+// ----------------------------------------------------------------------------
+// External DNS Variables
+// ----------------------------------------------------------------------------
 variable "apex_domain" {
   description = "Flag to enable or disable long term storage for logs"
```

Comment:
That seems wrong ;-)

Reply:
_create_vault_resources_, I still argue we should always create Vault. What's the alternative? 'local'? We really need to get away from it. I'd like to remove the mention of local secret storage asap from the docs. Not sure how much effort it is to remove from the code.